README.md

```diff
@@ -62,6 +62,7 @@ You can start working with this straight away with our [template][].
   - [`psql`](#psql)
   - [`inotify`](#inotify)
   - [`ptvsd`](#ptvsd)
```

Comment:
Please remove this section.

Although it still exists in old images, we don't want docs on deprecated stuff.